notebooks/Tutorial4_Client_side_processors.py

```diff
@@ -187,7 +190,9 @@
 
 # ### Relative differences with datetime variables
 #
-# The relative difference processor can also be used to express a date relative to another. To do so, it is required to use the `DatetimeProcessor` processor that will transform datetime variables into numeric values, enabling differences to be computed between date variables. Because the `DatetimeProcessor` has a post-process function, the data output by the avtarization pipeline will contain the datetime variables in their original format (i.e. as datetime rather than numeric values).
+# The relative difference processor can be used to express a date relative to another. To do so, it is required to use the `DatetimeProcessor` that will transform datetime variables into numeric values, by computing the differences between date variables.
+#
+# As the `DatetimeProcessor` has a post-processing function, the data produced by the avtarization pipeline will contain the datetime variables in their original format (i.e. as datetime values rather than numerical values).
```

Comment:
```suggestion
# As the `DatetimeProcessor` has a post-processing function, the data produced by the avatarization pipeline will contain the datetime variables in their original format (i.e. as datetime values rather than numerical values).
```